ownCloud/Key Commands/KeyCommands.swift

```diff
@@ -1016,13 +1042,90 @@ extension DisplayHostViewController {
 			shortcuts.append(previousObjectCommand)
 		}
 
+		if let core = core, let currentViewController = self.viewControllers?.first as? DisplayViewController, let item = currentViewController.item {
+			let actionsLocationCollaborate = OCExtensionLocation(ofType: .action, identifier: .keyboardShortcut)
+			let actionContextCollaborate = ActionContext(viewController: self, core: core, items: [item], location: actionsLocationCollaborate)
+			let actionsCollaborate = Action.sortedApplicableActions(for: actionContextCollaborate)
+
+			actionsCollaborate.forEach({
+				if let keyCommand = $0.actionExtension.keyCommand, let keyModifierFlags = $0.actionExtension.keyModifierFlags {
+					let actionCommand = UIKeyCommand(input: keyCommand, modifierFlags: keyModifierFlags, action: #selector(performMoreItemAction), discoverabilityTitle: $0.actionExtension.name)
+					shortcuts.append(actionCommand)
+				}
+			})
+		}
+
 		return shortcuts
 	}
 
 	override var canBecomeFirstResponder: Bool {
 		return true
 	}
 
+	@objc func performMoreItemAction(_ command : UIKeyCommand) {
+		if let core = core, let currentViewController = self.viewControllers?.first as? DisplayViewController, let item = currentViewController.item {
```

Comment:
@hosy try avoiding long lines for readability

Reply:
Fixed!